radicle-httpd/src/api/v1/projects.rs

```diff
@@ -778,7 +830,20 @@ mod routes {
                   "kind": "blob"
                 }
               ],
-              "lastCommit": null,
+              "lastCommit": {
```

Comment:
We should have this info in the output, not sure why it didn't.  This is also an example of the actual lastCommit value to replace a null.

Reply:
don't use plain hash, use `HEAD` like elsewhere

Reply:
Got it, will do.